### DISCUSSION TITLE
Ссылка на Шики

### DISCUSSION THREAD

**Discussion by forjosis** [cawa-93/anime-library#120] [Новые идеи]
Привет. Спасибо тебе за приложение!) Очень рад, что оно развивается.

Иногда бывает нужно вернуться на страницу аниме на Шики. Например, чтобы посмотреть на сэйю, который озвучивает персонажа и увидеть его роли. Или проверить другие тайтлы франшизы.

В браузерном расширении есть кнопка «Открыть на Шикимори». Возможно, что в приложении это тоже нужно)

**Comment by cawa-93** [comment]
С главной страницы можно нажать средней кнопкой или `Ctrl+Click` по постеру. Но это работает только с постерами в коллекциях.

**Reply by forjosis** [reply]
О, точно, не знал, что так можно. Спасибо)
